libraries/adaptive-expressions/src/expressionFunctions.ts

```diff
@@ -1005,16 +1008,31 @@ export class ExpressionFunctions {
         return {value: result, error};
     }
 
-    private static extractElement(expression: Expression, state: MemoryInterface): { value: any; error: string } {
+    private static wrapGetValue(state: MemoryInterface, path: string, options: Options): any {
+        let result = state.getValue(path);
+        if (result !== undefined) {
```

Comment:
do we only to this for "undefined", how about "null", i guess i'm not sure the convention is null or undefined

Reply:
> do we only to this for "undefined", how about "null", i guess i'm not sure the convention is null or undefined

will change to if(result), so it can handle both undefined and null